opm/autodiff/FlowMainEbos.hpp

```diff
@@ -700,6 +777,9 @@ namespace Opm
         Grid& grid()
         { return ebosSimulator_->gridManager().grid(); }
 
+        const Grid& globalGrid()
+        { return *globalGrid_; }
```

Comment:
BTW: I used `ebosSimulator->equilGrid()` before. But after the constructor has run this returns a dangling reference because the pointer to the grid is already deleted. Dangerous.